modules/network-peering/variables.tf

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "prefix" {
+  description = "Name prefix for the network peerings"
+  type        = string
+  default     = "network-peering"
+}
+
+variable "local_network" {
+  description = "Resource link of the network to add a peering to."
+  type        = string
+}
+
+variable "peer_network" {
+  description = "Resource link of the peer network."
+  type        = string
+}
+
+variable "peer_custom_routes" {
```

Comment:
Could we call these `export_peer_custom_routes` and `export_local_custom_routes` for clarity?

Reply:
Had a lenghty discussion yesterday brainstorming names, and while I'm fine with any reasonable name that would also be my preference. ;)

Reply:
Done!